app/views/users/_tabs.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="row">
+  <div class="col-xs-6">
+    <ul class="nav nav-tabs">
+      <li role="presentation" class="<%= "active" if params[:controller] == "users" %>"><a href="<%= users_path %>">Users</a></li>
```

Comment:
Great, let's pull it to a helper now! We are using similar conditionals in other places in master, but checking other things: 
_category_nav.html.erb: ```<li class="<%= @category == category ? "active" : "" %>">```
_navbar.html.erb: ```<li class="<%= "active" if active == "organizations" %>">```

Maybe something more generic, like ```active_class(base, to_compare)``` (I'm sure you'll come up with better param names :) )